PSICompile/ILCodeGen.cs

```diff
@@ -49,14 +49,15 @@ public class ILCodeGen : Visitor {
 
    public override void Visit (NAssignStmt a) {
       a.Expr.Accept (this);
-      StoreVar (a.Name);
+      ReadWriteVar (a.Name);
    }
 
-   void StoreVar (Token name) {
+   void ReadWriteVar (Token name, bool read = false) {

```

Comment:
Using a default value for **read** makes it difficult to understand when this function is being used in the code. Suppowe you just write `ReadWriteVar` in the code - how do I know if you are going to read or write the variable - that becomes clear only when I look at the function declaration. 

Alternative:
Have this as only a helper method that is never called from the code directly, and instead implement two one-liners that you use in the code:
```
void LoadVar (Token name) => ReadWriteVar (name, true);
void StoreVar (Token name) => ReadWriteVar (name, false);
```

Reply:
Changes applied